src/WallToolPaths.h

```diff
@@ -76,6 +76,13 @@ class WallToolPaths
      */
     static bool removeEmptyToolPaths(VariableWidthPaths& toolpaths);
 
+    /*!
+     * Simplifies The generated toolpaths
```

Comment:
A little bit more information about this would be nice. I understand that calling this function will simplify the generated toolpaths because that's the name of the function ;) 

Something like: "This will simplify the toolpath by calling the simplify on every line in the toolpath using the provideded settings" Would make more sense to me.